packages/@react-aria/tabs/docs/useTabList.mdx

```diff
@@ -319,7 +319,8 @@ tab selection.
 
 ### Orientation
 
-By default, tabs are horizontally oriented. The `orientation` prop can be set to `vertical` to change this. This affects keyboard navigation. You are responsible for styling your tabs accordingly.
+By default, tabs are horizontally oriented. The `orientation` prop can be set to `vertical` to change this. This does not affect keyboard navigation. Regardless of orientation, both ArrowUp and ArrowLeft navigate to the previous tab while ArrowDown and ArrowRight navigate to the next tab. Note that in rtl ArrowLeft and ArrowRight actions are flipped in `horizontal` orientation. 
+You are responsible for styling your tabs accordingly.
```

Comment:
```suggestion
By default, tabs are horizontally oriented. The `orientation` prop can be set to `vertical` to change this. This does not affects keyboard navigation. You are responsible for styling your tabs accordingly.
```

Mind omitting the details of the keyboard navigation here and in the other docs pages? IMO we can add those details once we settle on the correct RTL behavior

Reply:
Updated the docs to match your suggestion.